jac-uk/digital-platform
# Feature/1165 Convert functions from CommonJS to ESM

## What's included?
Closes #1165 

- Add `"type": "module"` to `package.json`.
- Replace the CommonJS syntax `require/module.exports` with the ESM syntax `import/export`.
- Update ESLint and Mocha configuration.

## How to test?

Run the following commands and check if all tests pass:
```
npm run test:rules
npm run test:functions
npm run test:actions
npm run test:shared
```
